_config.yml

```diff
@@ -159,6 +159,10 @@ contentful:
     query: "fields.exclude_from_crossroads[ne]=true"
     map:
       date: published_at
+  category:
+    query: "fields.collections.sys.id[nin]=3F1707Eeiu50rMaxCf6wNN,3X7oFhglWTJGXy3TD5u5br,6XwgUaVRdsKzXrncprwJMt,3E8JlwCIG49c00bU7MXEuL"
+  collection: 
+    query: "fields.slug[ne]=isgodreal-com"
```

Comment:
This seems a little brittle to me. Is there anyway to query for categories that have no collection assigned at all?

Reply:
If I understand the question correctly, yes, this query string already INCLUDES categories with no collection assigned. This query includes all categories except those with collections ids of IGR.

Reply:
For category, could you use what we've done for pages?

```
fields.collections[exists]=false
```

For collection, why bring them in at all? All we're using them for is the query. I think we could actually ignore them entirely and we should still be fine.